src/qtgui/iq_tool.h

```diff
@@ -85,6 +85,8 @@ private slots:
     void refreshDir(void);
     void refreshTimeWidgets(void);
     void parseFileName(const QString &filename);
+    void switchControlsState(bool recording, bool playback);
+
```

Comment:
NIT: Mey-be unintentionel extra new line? For this project there is no AFAIK code style guide, but usually one blank line, if any, is the convention.

Consider removel of the extra blank line, pls.

Reply:
Updated.
Thanks for suggestions.